solutions/swb-app/src/dataSets/projectAccessRequestParser.ts

```diff
@@ -12,7 +12,7 @@ export const ProjectAccessRequestParser = z
     authenticatedUser: AuthenticatedUserParser,
     dataSetId: z.string(),
     projectId: z.string(),
-    accessLevel: z.enum(['read-write', 'read-only'])
+    accessLevel: z.enum(['read-write', 'read-only']).optional()
```

Comment:
Is there a default? What happens when nothing is passed?

Reply:
There is no default. The value is only required when you're adding a project, not when removing one, and we check to make sure we have the value when we need it.